pages.fr/common/apm.md

```diff
@@ -0,0 +1,17 @@
+# apm
+
+> Manageur de paquets pour Atom.
+> Voir `atom`.
+> Plus d'informations : <https://github.com/atom/apm>.
+
+- Installe un paquet depuis http://atom.io/packages ou un thème depuis http://atom.io/themes :
+
+`apm install {{nom_du_paquet}}`
+
+- Supprime un paquet ou un thème :
+
+`apm remove {{nom_du_paquet}}`
+
+- Mets à jour un paquet ou un thème :
```

Comment:
```suggestion
- Met à jour un paquet ou un thème :
```

Reply:
same response as this PR ^^" 

https://github.com/tldr-pages/tldr/pull/8225#discussion_r925182444

Reply:
présent de l'indicatif explains better what a command does, at least in Spanish. I think this apply to French too.

Reply:
Both are ok in french, but depends on the context and not the actual form that nicokosi ask.

If you want to use the present de l'indicatif you must know that's a tense where you need to identify a subject before using a verb. (e, g j'aime les pommes (I like apples)). And you can't remove the `I` (`j'`) because you loose the `who`

Since we give implicitly an order to a machine (the command) then you don't need to specify the subject, BUT you must use imperative for the verb.

So if we want to use the present de l'indicatif, a correct sentence would be `il met à jour un paquet ou un thème :`
(it upgrades ....)

I hope my explanation are clear ^^" wdyt @nicokosi  ?

Reply:
Not sure to understand everything. :innocent:

I agree with: :ok_hand: 
> Both are ok in french,

IMHO, the subject is implicitly `apm` and indicative / third person is better. That's why I suggest "_met_" for "_(apm) met_" (indicative present, third person like "it"). :smiley_cat: